tests/lib/testers/rule-tester.js

```diff
@@ -636,4 +636,35 @@ describe("RuleTester", () => {
             });
         }, "Rule should not modify AST.");
     });
+
+    it("should throw an error if no test scenarios given", () => {
+        assert.throws(() => {
+            ruleTester.run("foo", require("../../fixtures/testers/rule-tester/modify-ast-at-last"));
+        }, "Test Scenarios for rule foo : Could not find test scenario object");
+    });
+
+    it("should throw an error if no valid test scenario object given", () => {
```

Comment:
Nitpick: The use of "valid" here is a little confusing because in this context it's easy to mistake it for the `valid`/`invalid` tests in our tests files rather than if the test scenario object itself is valid. Wonder if another term would make that clearer?